src/main/java/com/projetonetflix/projetonetflix/model/entity/Director.java

```diff
@@ -0,0 +1,52 @@
+package com.projetonetflix.projetonetflix.model.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "DIRECTOR")
+public class Director {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "IDT_DIRECTOR")
+    private Long id;
+
+    @Column(name = "NAM_DIRECTOR")
+    private String namDirector;
+
+    @Column(name = "NUM_AGE_DIRECTOR")
+    private int numAgeDirector;
```

Comment:
Faz sentido ser primitivo? Não sei como está a tabela, mas se a tabela é not null, aqui até faz sentido ele ser primitivo, mas vc corre o risco de salvar idade 0 e ficar uma informação estranha.
Acho que se fosse Integer, ficaria algo mais natural interpretar null como desconhecido, já que 0 não necessariamente é invalido. E caso vc queira passar a info de que é desconhecida, vc teria que manipular o dado para algo como -1, que é uma idade inválida.

Reply:
Beleza Go entendi, é que na minha cabeça eu fiquei assim, só eu vou usar ai eu sei como funciona kkkk